assets/tailwind.config.js

```diff
@@ -29,8 +29,13 @@ module.exports = {
   theme: {
     extend: {
       colors: {
-        ...colors
-      }
+        ...colors,
+        "gray-lighter": "#b0b5c0",
+        "gray-light": "#788093",
```

Comment:
These colors should be in the format `hue-shade`.

Reply:
What does this mean?

I couldn't find those hex codes elsewhere in dotcom or in metro, and I named them so that the names in Figma and the names in our code would line up.

Reply:
All other colors take the form of `hue-shade`. E.g., `blue-500`, `red-300`.

https://github.com/anthonyshull/mbta_metro/blob/main/assets/js/styles/colors.js

Reply:
The names in Figma are that way because that's what Dotcom uses currently. 

These are actually captured as Sass variables ([link](https://github.com/mbta/dotcom/blob/603367ef55285c1ed8ad131f0d601c3e46b993d7/assets/css/_colors.scss#L20-L24)) but did not yet have corresponding utility classes. This adds them! I think they're useful to have until we derive everything from the design system.